core/src/mindustry/input/DesktopInput.java

```diff
@@ -485,6 +486,14 @@ void pollInput(){
             schemY = -1;
         }
 
+        if(Core.input.keyTap(Binding.skipWaveCall)){
+            if(net.client() && player.admin && logic.canSkipWave()){
+                Call.adminRequest(player, Packets.AdminAction.wave);
```

Comment:
Inner class should be referenced directly as AdminAction instead of qualifying it.